pkg/dnsmasq/const.go

```diff
@@ -0,0 +1,29 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dnsmasq
+
+const (
+	// ServiceName -
+	ServiceName = "dnsmasq"
+	// ServiceAccount -
+	ServiceAccount = "infra-operator-dnsmasq"
```

Comment:
How is this getting used. And are we generating an account for this in each namespace then?

Reply:
good catch, thanks. missed this part to integrate with the new rbac way. will fix it

Reply:
done